subsys/usb/usb_c/Kconfig

```diff
@@ -39,6 +39,16 @@ config BUILD_OUTPUT_VIF
 	help
 	  Generate XML file containing VIF policies during project build.
 
+config GENVIF_INPUT_VIF_XML_PATH
+	string "Path to input VIF file"
+	depends on BUILD_OUTPUT_VIF
+	help
+	  Pass custom path of input VIF XML file residing in application source
+	  directory to GenVIF script. Consider application source directory as source
+	  directory. If this config option is not configured, the script looks for a
```

Comment:
I reword with something like "Path of the input VIF XML file relative to the application source directory."

Reply:
I updated the description in new revision